tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -68,16 +68,16 @@ internal static partial class ConfigurationKeys
         public const string TraceEnabled = "DD_TRACE_ENABLED";
 
         /// <summary>
-        /// Configuration key for enabling Standalone ASM, thus disabling APM tracing and its billing.
+        /// Configuration key for enabling or disabling the Tracer's debug mode.
         /// Default is value is false (disabled).
         /// </summary>
-        public const string AppsecStandaloneEnabled = "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED";
+        public const string DebugEnabled = "DD_TRACE_DEBUG";
 
         /// <summary>
-        /// Configuration key for enabling or disabling the Tracer's debug mode.
-        /// Default is value is false (disabled).
+        /// Configuration key for enabling or disabling the generation of APM traces.
+        /// Default is value is true (enabled).
         /// </summary>
-        public const string DebugEnabled = "DD_TRACE_DEBUG";
+        public const string ApmTracingEnabled = "DD_APM_TRACING_ENABLED";
```

Comment:
woah woah woah... Really? Has this been cleared with the SDK team? Because this doesn't seem like a config that ASM should be unilaterally introducing 😅

Reply:
> Has this been cleared with the SDK team?

Yes. See RFC link in PR description. (thanks for checking!)